tests/http/test_client.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import respx
+from httpx import ConnectTimeout, Response, codes
+
+from mpt_api_client.http.client import MPTClient
+
+API_TOKEN = "test-token"
+API_URL = "https://api.example.com"
+
+
+@pytest.fixture
+def mpt_client():
+    return MPTClient(base_url=API_URL, api_token=API_TOKEN)
+
+
+def test_mpt_client_initialization():
+    client = MPTClient(base_url=API_URL, api_token=API_TOKEN)
+
+    assert client.api_token == API_TOKEN
+    assert client.base_url == API_URL
+
+
+def test_mpt_client_headers():
+    client = MPTClient(base_url=API_URL, api_token=API_TOKEN)
+
+    assert client.headers["Authorization"] == "Bearer test-token"
+    assert client.headers["User-Agent"] == "swo-marketplace-client/1.0"
+
+
+@respx.mock
+def test_mock_call_success(mpt_client: MPTClient):
```

Comment:
Since we are ignoring running mypy in the test folder, typing here won't be checked so we shouldn't use it.